modules/uncertainty/R/run.sensitivity.analysis.R

```diff
@@ -133,7 +133,7 @@ run.sensitivity.analysis <- function(settings,plot=TRUE, ensemble.id=NULL, varia
             pdf(fname, height = 12, width = 9)
             ## arrange plots  http://stackoverflow.com/q/10706753/199217
             ncol <- floor(sqrt(length(sensitivity.plots)))
-            print(do.call("grid.arrange", c(sensitivity.plots, ncol=ncol)))
+            print(do.call("gridExtra::grid.arrange", c(sensitivity.plots, ncol=ncol)))
```

Comment:
```suggestion
            print(do.call(gridExtra::grid.arrange, c(sensitivity.plots, ncol=ncol)))
```

(I don't know why do.call accepts non-namespaced function names as character but namespaced ones only as symbols, but at least the fix is easy to implement 🤷‍♂️)